test/cli_test.rb

```diff
@@ -0,0 +1,124 @@
+require 'test_helper'
+require 'shopify_cli/cli'
+require 'fileutils'
+
+class CliTest < Test::Unit::TestCase
+  def setup
+    @test_home = File.join(File.expand_path(File.dirname(__FILE__)), 'files', 'home')
+    @shop_config_dir = File.join(@test_home, '.shopify', 'shops')
+    @default_symlink = File.join(@shop_config_dir, 'default')
+    `rm -rf #{@test_home}`
+    ENV['HOME'] = @test_home
+    @cli = ShopifyCLI::Cli.new
+
+    FileUtils.mkdir_p(@shop_config_dir)
+    File.open(config_file('foo'), 'w') do |file|
+      file.puts valid_options.merge('domain' => 'foo.myshopify.com').to_yaml
+    end
+    File.symlink(config_file('foo'), @default_symlink)
+    File.open(config_file('bar'), 'w') do |file|
+      file.puts valid_options.merge('domain' => 'bar.myshopify.com').to_yaml
+    end
+  end
+
+  def teardown
+    `rm -rf #{@test_home}`
```

Comment:
this can have hilarious consequences if `@test_home` for some reason became say `~/`. Is a bir paranoid but I'd probs make supre sure that the value of @test_home is sensible.